inst/include/cpp11/R.hpp

```diff
@@ -48,8 +49,17 @@ template <typename T>
 inline T na();
 
 template <typename T>
-inline bool is_na(const T& value) {
+inline typename std::enable_if<!std::is_same<typename std::decay<T>::type, double>::value,
```

Comment:
I trust this approach more than an unconstrained `template <typename T>` followed by a fully specialized `is_na<double>`. 

And also that means all of `is_na()` is in this one file.